internal/junitxml/report_test.go

```diff
@@ -19,8 +22,34 @@ func TestWrite(t *testing.T) {
 	out := new(bytes.Buffer)
 	exec := createExecution(t)
 
+	tpl, err := template.ParseFiles("testdata/junitxml-report.golden.tpl")
+	assert.NilError(t, err)
+	f, err := os.Create("testdata/junitxml-report.golden")
+	assert.NilError(t, err)
```

Comment:
Instead of a templated expected value, I think it is easier to maintain when we patch the expected value. See `Config.customTimestamp` for an example of that. We should be able to specify a value for `hostname` in the test, so that our expected value doesn't change between hosts.

I believe timestamp is already patched, so everything else shouldn't need a template.

That change should allow us to remove the template and go back to a regular golden file.

Reply:
Thank you for the example. I changed the test.